deps.edn

```diff
@@ -36,15 +36,15 @@
                                              :exclusions  [it.unimi.dsi/fastutil
                                                            org.slf4j/slf4j-api]}
   com.draines/postal                        {:mvn/version "2.0.5"}              ; SMTP library
-  com.github.seancorfield/honeysql          {:mvn/version "2.5.1103"}           ; Honey SQL 2. SQL generation from Clojure data maps
+  com.github.seancorfield/honeysql          {:mvn/version "2.6.1126"}           ; Honey SQL 2. SQL generation from Clojure data maps
   com.github.seancorfield/next.jdbc         {:git/url "https://github.com/seancorfield/next-jdbc.git"
                                              :sha     "abd926cde3730087d3729dcea0ce0ab7ef1194f2"} ; for https://github.com/seancorfield/next-jdbc/issues/245; when this makes it to a release we can switch to that.
```

Comment:
Don't want to update this one? :) The fix is merged already, but last time I've tried that (updating next.jdbc and honeysql) I broke master - in 10cdb49db4d5105962e418c5f23f9bdae3309450

Reply:
ah, good call. Confirmed that it's in the newer version, so we should be good tog o

Reply:
Beware: `be-linter-cloverage` step runs only on master, and that was the one broken when I tried to update this back in November. So maybe it still makes sense to have `next.jdbc` as a separate PR?

Reply:
fair enough, will tag you in the separate PR in a sec

Reply:
https://github.com/metabase/metabase/pull/40452